aerial_robot_control/include/aerial_robot_control/flight_navigation.h

```diff
@@ -441,11 +441,15 @@ namespace aerial_robot_navigation
         {
           if(msg->data == 0)
             {
+              setTargetXyFromCurrentState();
+              target_acc_.setValue(0, 0, 0);
```

Comment:
`target_vel_` should be also reset here?

Reply:
@kanke-hayata 

I also agree with @chibi314 . Please add `target_vel_.setValue(0, 0, 0);` here